app/assets/javascripts/ckeditor/plugins/cortex_media_insert/plugin.js

```diff
@@ -1,19 +1,19 @@
-(function (global) { 
+(function (global) {
   'use strict';
 
   global.CKEDITOR.plugins.add('cortex_media_insert', {
     icons: 'media',
     init: function (editor) {
       editor.addCommand('insertMedia', {
         exec: function (editor) {
-          window.MODALS.featured.open();
+          window.MODALS.wysiwyg.open();
```

Comment:
Featured, Tile and WYSIWYG are all 3 separate, temporarily hardcoded popups in an Employer Blog CRUD page. This plugin was opening the wrong popup (featured) for CKEditor, when it should have been opening the WYSIWYG popup - hence the change here. The featured/wysiwyg/tile objecst in this chain are dynamically derived from `data` attributes on each of our 3 popup HTML elements.